ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml

```diff
@@ -40,8 +40,8 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: rosa-aws-sts-hypershift
   timeout: 5h0m0s
-- always_run: false
-  as: perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodic
+- as: perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodi
+  cron: 0 20 * * 5
```

Comment:
Just once a week?

Reply:
yes, for self managed we can run more often if needed, but for managed once a week should suffice. I confirmed with Roshni. 
If we ever find once a week is not enough, we can bump it up.

Reply:
We can run these rosa-hcp jobs once weekly this should be good to start with.